cypress/integration/integration_spec.js

```diff
@@ -12,1190 +12,1091 @@ function acceptCookieConsent() {
     return cy.get('.cookie-consent button').click();
 }
 
-describe('server smoke tests', function() {
```

Comment:
We mostly had a single `it` in each `describe` block so we can reduce nesting by removing these